### PR TITLE
Feature swagger properties table operations

### PR DESCRIPTION
**Pull request by Blackbaud-LoriWeiss**
Refactoring current entity reference to create a new directive for generating property tables for the endpoint reference page.

### CODE REVIEW

**Comment by Blackbaud-LoriWeiss** [comment]
For testing, I replaced my local Constituent Entity reference with the following snippet, and used various Operation Ids.

```
<div ng-app="entityReferenceApp">
  <bb-operation-entity-table
    swagger-url="https://developer.sky.blackbaud.com/docs/services/56b76470069a0509c8f1c5b3/export?DocumentFormat=Swagger&ApiName=Constituent"
    operation-id="578d324da9db950becc619fe"
    base-link-url='http://localhost:4000/api/entity-reference/fundraising'>
  </bb-operation-entity-table>
</div>```

File: static/assets/js/entity-reference/app.1.2.js [hunk 1]
```diff
@@ -208,4 +127,166 @@
...
+            $http.get(self.swaggerUrl)
+              .then(handleSuccess, handleError)
+              .finally(function() { bbWait.endPageWait(); });
+        }
+
+        function handleSuccess(response) {
+            handleSwaggerResponseData(response.data);
+        }
+
+        function handleSwaggerResponseData(swagger) {
```
[the remark this answers is not in the document]
Comment (Blackbaud-LoriWeiss) [reply static/assets/js/entity-reference/app.1.2.js]: Yeah, I actually debated that myself, but it was already like that.... I can do that...

[the remark this answers is not in the document]
Comment (Blackbaud-LoriWeiss) [reply static/assets/js/entity-reference/app.1.2.js]: Why do you say indents should be two spaces?

**Comment by Blackbaud-LoriWeiss** [comment]
@blackbaud-johnly can we merge this, if the builds are working?

**Comment by Blackbaud-LoriWeiss** [comment]
@Blackbaud-LindseyRix are you around?

**Comment by blackbaud-johnly** [comment]
@Blackbaud-LoriWeiss, I didn't notice this with all the other automated emails in my inbox this afternoon. I can merge this in.

**Pull request merged**